### PR TITLE
Incoming call ring tones & desktop notifications

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for:
- https://github.com/mattermost/mattermost/pull/23444
- https://github.com/mattermost/mattermost-plugin-calls/pull/449

### CODE REVIEW

File: source/configure/plugins-configuration-settings.rst [hunk 1]
```diff
@@ -915,6 +915,23 @@ Call recording quality
...
+
+.. config:setting:: plugins-enablecallringing
+  :displayname: Enable call ringing (Plugins - Calls)
+  :systemconsole: Plugins > Calls
+  :configjson: N/A
+  :environment: N/A
+  :description: Enable or disable incoming call desktop alerts and ringing notifications
+
++--------------------------------------------------------------------------+--------------------------------------------------+
+| - **true**: Ringing functionality is enabled. Direct and group message   | - System Config path: **Plugins > Calls**        |
```
Comment (cwarnermm) [inline source/configure/plugins-configuration-settings.rst:929]: @cpoile - Does this new Enable Ringing config setting have an equivalent ``config.json`` and environment variable value to include in the docs?

File: source/channels/channels-settings.rst [hunk 2]
```diff
@@ -41,6 +41,13 @@ Notification sound
 
 When desktop app notifications are set to **For all activity** or **Only for mentions and direct messages**, you can configure Mattermost to notify you using a sound. Chooose from one of 6 preset notification sounds. Notification sounds are available on Chrome, Edge, Firefox, and Safari in addition to the Mattermost Desktop App.
 
+Notification sound for incoming calls (Beta)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When desktop app notifications are set to **For all activity** or **Only for mentions and direct messages**, you can configure Mattermost to alert you to incoming calls through direct or group messages with a specific ring tone and a desktop notification, unless the system admin has `disabled your ability to do so </plugins-configuration-settings.html#enable-call-ringing-beta>`__.
```
Comment (cpoile) [inline source/channels/channels-settings.rst:47]: We might want to mention that this feature is available starting with MM v8.0 and Calls v0.17.0 -- although it's prepackaged, we might want to set expectations.

File: source/channels/make-calls.rst [hunk 3]
```diff
@@ -113,6 +113,11 @@ To record a call:
 Frequently asked questions
 --------------------------
 
+Can I set a ring tone for incoming calls?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes! From Mattermost v8.0, desktop app users can go to **Settings > Notifications > Desktop Notifications** to enable Mattermost to alert you to incoming calls through direct or group messages with a specific ring tone and a desktop notification, unless the system admin has `disabled your ability to do so </plugins-configuration-settings.html#enable-call-ringing-beta>`__.
```
Comment (cpoile) [inline source/channels/make-calls.rst:119]: Maybe:

```suggestion
Yes! From Mattermost v8.0 and Calls v0.17.0, desktop app and webapp users can go to **Settings > Notifications > Desktop Notifications** to enable Mattermost to alert you to incoming calls through direct or group messages with a specific ring tone and a desktop notification, unless the system admin has `disabled your ability to do so </plugins-configuration-settings.html#enable-call-ringing-beta>`__.
```

File: source/configure/plugins-configuration-settings.rst [hunk 4]
```diff
@@ -915,6 +915,23 @@ Call recording quality
...
+|   participants receive a desktop app alert and a ringing notification    | - ``config.json`` setting: N/A                   |
```
Comment (cpoile) [inline source/configure/plugins-configuration-settings.rst:930]:
```suggestion
|   participants receive a desktop app alert and a ringing notification    | - ``config.json`` setting: ``PluginSettings.Plugins.com.mattermost.calls.enableringing``                   |
```

File: source/configure/plugins-configuration-settings.rst (hunk 1 above)
Comment (cpoile) [reply source/configure/plugins-configuration-settings.rst:929]: Yep, added it above and below. No environment variable.

**Review by cpoile — APPROVED** [review]
Thanks Carrie, added the info. Thank you!

**Pull request merged**